cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -220,6 +220,11 @@ func (uc *UpstreamController) dispatchMessage() {
 		}
 	}
 }
+
+type RuleStatus struct {
```

Comment:
Why do we need this struct?

Reply:
k8s patch RuleStatus must specify the object of status, may be t this structure is placed in the wrong place